packages/components/src/Chip/Chip.css

```diff
@@ -129,31 +128,32 @@
 */
 
 .subtle:focus-visible {
-  --chip--gradient-color-variation--hover: var(--color-surface);
+  /* this one looks suspicious */
+  /* why does this not match the hover style? */
```

Comment:
Were you able to find the answer? In the [original Figma](https://www.figma.com/design/rIIhulZvcp9M82lNOCGv16/branch/S6J5803Pjmy4VvpsQx0o8S/Product%2FOnline?node-id=18944-3858&node-type=frame&m=dev) it looks like the bg colour in hover and focus states should be the same. I didn't find any comments in previous PRs that modified this file that would explain why "subtle:focus-visible" is getting a different styling compared to ALL other chip variants.

I looked around (A LOT actually) and from what I can tell, there might be chance that `.subtle:focus-visible` background-color was set incorrectly 11 months ago in [this PR](https://github.com/GetJobber/atlantis/pull/1593/files#diff-5dd56e5b5ba5cd1980ca822ded40f143927988157353a03ea46d24e70e41d84fR111-R112) 

1. It looks like the focus state wasn't styled in the [original PR by Scott](https://github.com/GetJobber/atlantis/pull/1571/files#diff-5dd56e5b5ba5cd1980ca822ded40f143927988157353a03ea46d24e70e41d84fR89-R96).
2. In the following PR by Zak (the one I mentioned above), there was a new CSS rule added to style `.subtle:focus`, however, for some reason the background-color CSS variable was assigned a value different from the one in `.subtle:hover` block.

So I think that the incorrect colour could have been set in [this commit](https://github.com/GetJobber/atlantis/pull/1593/commits/fe7d93738b61725d6bc0c5f8a229b6a3a772faa0#diff-5dd56e5b5ba5cd1980ca822ded40f143927988157353a03ea46d24e70e41d84fR108-R111) and was potentially missed during testing.
My opinion is that the hover and focus bg colours should be the same. Perhaps, it would be great if @chris-at-jobber provided his opinion on this.

Reply:
yeah it was 2 things. we made some changes 11 months ago that might have introduced some divergence, though I think that might have been intentional, and then with the separate-chip-from-chips PR, the focus visible broke altogether and we can't really use that selector when the Chip is rendered as a `div`, which is exactly what happens in multi/single select chips since the `onClick` isn't actually passed to the Chip, it's on some element above it.

https://github.com/GetJobber/atlantis/pull/2035#issuecomment-2369910585

Reply:
@chris-at-jobber how important would you say this is to have the subtle focus state match the hover state?

fixing it for Chip, isn't too bad. we just choose which appearance we want and apply it in both cases

fixing it for Chips is a bit more. I have a not that ideal solution with duplicating CSS.

if we're not overly worried about it, I can leave it be.

Reply:
It would be preferred, I feel like a bit of extra CSS is worth standardizing the experience

Reply:
ok, got them matching for normal Chip

then added the less-than-ideal copied styles to Chips > InternalChip.css which will keep them looking the same

the TL;DR of the problem is that we have `focus-visible` selectors and styles in `Chip.css` however in multi and single select `Chips` it's actually a hidden `input` that we are focusing, so it doesn't trigger the focus styles of `Chip`

previously, and now with the update, we apply the same focus style via a sibling selector in `InternalChip.css` but it is a second place with the same definition 😿